docs/tutorial.rst

```diff
@@ -102,11 +104,415 @@ mlf-core is now configured and ready to be used!
+
+    entry_points:
+    main:
+        parameters:
+        max_epochs: {type: int, default: 5}
+        gpus: {type: int, default: 0}
+        accelerator: {type str, default: "None"}
+        lr: {type: float, default: 0.01}
+        general-seed: {type: int, default: 0}
+        pytorch-seed: {type: int, default: 0}
+        command: |
+            python project_name/project_name.py \
+                --max_epochs {max_epochs} \
+                --gpus {gpus} \
+                --accelerator {accelerator} \
+                --lr {lr} \
+                --general-seed {general-seed} \
+                --pytorch-seed {pytorch-seed}
+
+mlf-core projects by default run with Docker. If you prefer to run your project with Conda you need to comment in ``conda_env`` and comment out
+``docker_env`` and its associated configuration. We are currently working on easing this switching, but for now it is a MLflow limitation.
+The ``image`` by default points to the Docker image build on Github Packages which automatically happens on project creation.
+Moreover, all runs mount the data directory in the root folder of the project to ``/data`` inside the container.
+Therefore, you need to ensure that your data either resides in the data folder of your project or adapt the mounted volumes to include your training data.
+mlf-core also presets environment variables required for deterministic machine learning. Do not modify them without an exceptional reason.
+Finally, the project_name.py file is set as an entry point and all parameters are defined and passed with MLflow.
+
+Dockerfile
+~~~~~~~~~~~~
+
+The Dockerfile usually does not need to be adapted.
+It is based on a custom mlf-core base container which provides CUDA, Conda and other utilities.
+
+.. code-block::
+
+    FROM mlfcore/base:1.2.0
+
+    # Install the conda environment
+    COPY environment.yml .
+    RUN conda env create -f environment.yml && conda clean -a
+
+    # Activate the environment
+    RUN echo "source activate exploding_springfield" >> ~/.bashrc
+    ENV PATH /home/user/miniconda/envs/exploding_springfield/bin:$PATH
+
+    # Dump the details of the installed packages to a file for posterity
+    RUN conda env export --name exploding_springfield > exploding_springfield_environment.yml
+
+The Docker container simply uses the environment.yml file to create a Conda environment and activates it.
+You can find the base container definitions in the `mlf-core containers repository <https://github.com/mlf-core/containers>`_.
+
+environment.yml
+~~~~~~~~~~~~~~~~
+
+The ``environment.yml`` file is used for both, running the mlf-core project with Conda, and for creating the Conda environment inside the Docker container.
+Therefore you only need to specify your dependencies once in this file.
+Try to always define all dependencies from Conda channels if possible and only add PyPI dependencies if a Conda version is not available.
```

Comment:
Maybe a short hint, that one should not blindly update the default versions here, because its not tested or verified to be always deterministic in case of an update?